From 1ebacc69cb9b9c2508b1e75cb23ef572cda51eec Mon Sep 17 00:00:00 2001
From: Alison Wang <b18965@freescale.com>
Date: Thu, 4 Aug 2011 09:59:47 +0800
Subject: [PATCH 33/52] Fix structure fsl_ssd1289_data definition bug for SSD1289 FB driver

Fix the bug that structure fsl_ssd1289_data definition depends on
DSPI enabled for SSD1289 framebuffer driver.

Signed-off-by: Alison Wang <b18965@freescale.com>
---
 arch/m68k/coldfire/m5441x/devices.c |   20 ++++++++++----------
 1 files changed, 10 insertions(+), 10 deletions(-)

--- a/arch/m68k/coldfire/m5441x/devices.c
+++ b/arch/m68k/coldfire/m5441x/devices.c
@@ -138,6 +138,16 @@ static struct platform_device nfc_device
 };
 #endif
 
+#if defined(CONFIG_FB_FSL_SSD1289) || defined(CONFIG_FB_FSL_SSD1289_MODULE)
+static struct fsl_ssd1289_fb_display fsl_ssd1289_data = {
+	.width		= 320,
+	.height		= 240,
+	.xres		= 320,
+	.yres		= 240,
+	.bpp		= 16,
+};
+#endif
+
 /*
  * DSPI
  */
@@ -197,16 +207,6 @@ static struct coldfire_dspi_chip at26df0
 };
 #endif
 
-#if defined(CONFIG_FB_FSL_SSD1289) || defined(CONFIG_FB_FSL_SSD1289_MODULE)
-static struct fsl_ssd1289_fb_display fsl_ssd1289_data = {
-	.width		= 320,
-	.height		= 240,
-	.xres		= 320,
-	.yres		= 240,
-	.bpp		= 16,
-};
-#endif
-
 #if defined(CONFIG_SSD1289_SPI_MODE)
 static struct coldfire_dspi_chip ssd1289_chip_info = {
 	.mode = SPI_MODE_0,
